app/src/main/java/com/simplecity/amp_library/playback/MediaManager.kt

```diff
@@ -0,0 +1,164 @@
+package com.simplecity.amp_library.playback
+
+import android.net.Uri
+import com.simplecity.amp_library.model.Album
+import com.simplecity.amp_library.model.AlbumArtist
+import com.simplecity.amp_library.model.Genre
+import com.simplecity.amp_library.model.Song
+import io.reactivex.Single
+
+/**
+ * Created by fxsalazar
+ * 01/05/2018.
+ */
+interface MediaManager {
+
+    interface Defs {
```

Comment:
This appears to be re-declared in the `MusicUtils` implementation of `MediaManager`